arm-eventhub/2017-04-01/swagger/EventHub.json

```diff
@@ -1661,6 +1666,71 @@
       ],
       "description": "Single item in List or Get Event Hub operation"
     },
+    "CaptureDescription": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "A value that indicates whether capture description is enabled. "
+        },
+        "encoding": {
+          "type": "string",
+          "description": "Enumerates the possible values for the encoding format of CaptureDescription.",
+          "enum": [
+            "Avro",
+            "AvroDeflate"
+          ],
+          "x-ms-enum": {
+            "name": "EncodingCaptureDescription",
+            "modelAsString": false
```

Comment:
Talked with Dev team, they are good with  enum and the values are not going to change in near future. we have similar implementation for other enums.